base/applications/regedit/main.h

```diff
@@ -27,7 +27,7 @@
 #define TREE_WINDOW     2002
 #define LIST_WINDOW     2003
 
-#define	SPLIT_WIDTH	5
+#define    SPLIT_WIDTH    5
```

Comment:
Wouldn't one space be better between #define and SPLIT_WIDTH ?

Reply:
Yes

Reply:
It's not consistent with other #define which only have a space and not a tab